src/main/kotlin/ApplicationConstants.kt

```diff
@@ -12,8 +12,8 @@ class ApplicationConstants {
     @Value("\${workday.average_training_days}")
     lateinit var averageTrainingDays: String
 
-    @Value("\${workday.average_holi_days}")
-    lateinit var averageHoliDays: String
+    @Value("\${workday.average_leave_days}")
+    lateinit var averageLeaveDays: String
```

Comment:
Dit moet ook avarageHoliDays blijven. Dit is een constante voor een berekening.

Reply:
Heb ik aangepast, inclusief in de .properties files